routes/truckRoutes.js

```diff
@@ -1 +1,105 @@
-// put your Express router code in here
\ No newline at end of file
+'use strict';
+// put your Express router code in here
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+var express = require('express');
+var foodTrucks = require('../trucks');
+
+var router = express.Router();
+
+var trucks = require('../trucks');
+var foodTypes = require('../trucks');
+var openTrucks = require('../trucks');
+var typeTrucks = require('../trucks');
+/* eslint-enable no-undef */
+
+// var serveStatic = express.static('public');
+
+var axeTruck = 'Fez';
+
+var openTruck = trucks.filterTrucksByDay('Saturday');
+var pizzaTruckList = trucks.filterTrucksByFoodType('pizza');
+var addTruck = trucks.addTruck();
+var deleteTruck = trucks.removeTruck(axeTruck);
+
+/*
+.use(function (request, response) {
+  console.log('Request URL: ', request.originalURL);
+})
+.use(function (request, response) {
+  console.log('Time: ', Date.now());
+})
+*/
+
+/* eslint-disable no-console */
+
+router.route('/')
+.get(function (request, response) {
+//   `/trucks` This route returns the list of all trucks in the module.
+// http://127.0.0.1:3000/trucks
+  var truckList = trucks.getTrucks();
+  // response.send(truckList);
+  response.status(200).json(trucks.getTrucks());
+})
+.get(function (request, response) {
```

Comment:
Not sure why this additional get route is here. The get route that appears before this will handle the request, meaning this code will never be reached.